style.css

```diff
@@ -85,3 +89,415 @@ p {
   width: 172px;
   list-style: none;
 }
+
+.myworks {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  width: 80%;
+  margin-bottom: 50px;
+}
+
+.vector {
+  border-radius: 5px;
+  border: 2px solid #36b37e;
+  background-color: #36b37e;
+  width: 50px;
+}
+
+.card {
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  grid-template-columns: 1fr;
+  background: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  width: 85%;
+  height: 474px;
+  margin: 10px;
+}
+
+.box {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  grid-area: 2/1;
+  background-color: white;
+  border: 1px solid white;
+  border-radius: 8px;
+  align-items: center;
+  list-style: none;
+  margin: 1px;
+  padding: 1px;
+}
+
+h4 {
+  font-family: inter, sans-serif;
```

Comment:
- [ ] [REQUIRED] For increasing 🆙 your chances in the job search 💼 after finishing microverse, we request that you follow coding best practices 💯  in all of your projects. One of these practices is **DRY** (**D**on't **R**epeact **Y**ourself). That means if you find yourself repeating 🔁 a specific line of code in multiple places, you have to find a way to only use it once 1️⃣. This way when you update ♻️  this line, you won't have to remember where else did you write that to update the other copies.

### You can check this in the [Html & Css Best Practices Page](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

![image](https://user-images.githubusercontent.com/60979926/164267978-27c9c5e1-37bb-4d2d-81ef-e7cd491bfbdb.png)

### In our case

you can easily use a `.font` class in multiple places or simply put the `font-family` property in the `body` selector and all other children elements going to inherit it by default.

- (your code)

```css
.h4 {
  /* ...code */
  font-family: inter, sans-serif;
  /* ...code */
}

.ror {
  /* ...code */
  font-family: inter, sans-serif;
  /* ...code */
}
```

- suggestion

```css
.font {
  font-family: inter, sans-serif;
}
```

> **Note**: Also, you can do the same thing with `display: flex` declaration.